server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -1795,4 +1798,17 @@ public void testReservesBytesDuringPeerRecoveryPhaseOne() throws Exception {
             .mapToLong(n -> n.getIndices().getStore().getReservedSize().getBytes()).sum(), equalTo(0L));
     }
 
+    private void assertGlobalCheckpointIsStableAndSyncedInAllNodes(String indexName, int shardId) throws Exception {
+        long maxSeqNo = NO_OPS_PERFORMED;
+        for (IndicesService indicesService : internalCluster().getDataNodeInstances(IndicesService.class)) {
+            IndexShard shard = indicesService.getShardOrNull(new ShardId(resolveIndex(indexName), shardId));
+            maxSeqNo = Math.max(maxSeqNo, shard.seqNoStats().getMaxSeqNo());
```

Comment:
If we're certain that indexing has finished by this point then the MSN should be equal across all copies (so we can get it from any of them, and maybe assert it's equal everywhere). If we're not certain that indexing has finished then the subsequent `assertBusy()` won't always work either.

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/75898/commits/20585d2aacbe324649fa273d039fc309d5ebc816